scip-java/src/main/scala/com/sourcegraph/scip_java/buildtools/BazelBuildTool.scala

```diff
@@ -13,15 +26,216 @@ class BazelBuildTool(index: IndexCommand) extends BuildTool("Bazel", index) {
   }
 
   override def generateScip(): Int = {
-    index
-      .app
-      .reporter
-      .error(
-        "The `scip-java index` command does not work with Bazel builds because Bazel " +
-          "requires custom configuration to be added to WORKSPACE and BUILD files. " +
-          "The scip-java repository contains an example WORKSPACE file to demonstrate " +
-          "how to use scip-java with Bazel https://github.com/sourcegraph/scip-java/blob/main/examples/bazel-example/WORKSPACE"
-      )
-    1
+    this.generateAspectFile() match {
+      case None =>
+        1
+      case Some(aspectLabel) =>
+        val buildCommand = ListBuffer.empty[String]
```

Comment:
nit: I think it's more idiomatic to use `List.newBuilder[String]` if you don't access the full accumulated result _before_ you finish appending to it.

Reply:
As mentioned in https://github.com/sourcegraph/scip-java/pull/610/files#r1262341973 I'm not sure this has to be a mutable thing at all, seems like appending one by one is not strictly necessary in this application.